JL.Core/Mining/MiningUtils.cs

```diff
@@ -440,6 +561,76 @@ public static async Task MineToFile(LookupResult lookupResult, string currentTex
         Utils.Logger.Information("Mined {PrimarySpelling}", lookupResult.PrimarySpelling);
     }
 
+    public static async ValueTask<bool[]?> CheckDuplicates(List<LookupResult> lookupResults, string currentText, int currentCharPosition)
+    {
+        List<Note> notes = [];
+        List<int> positions = [];
+        bool[] results = new bool[lookupResults.Count];
+        for (int i = 0; i < lookupResults.Count; i++)
+        {
+            LookupResult lookupResult = lookupResults[i];
+
+            Dictionary<MineType, AnkiConfig>? ankiConfigDict = await AnkiConfig.ReadAnkiConfig().ConfigureAwait(false);
+            if (ankiConfigDict is null)
+            {
+                continue;
+            }
```

Comment:
```suggestion
            Dictionary<MineType, AnkiConfig>? ankiConfigDict = await AnkiConfig.ReadAnkiConfig().ConfigureAwait(false);
            if (ankiConfigDict is null)
            {
                return null;
            }
```

This should be done at the beginning of the method, not in the for loop